hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/ConnectionContext.java

```diff
@@ -42,7 +44,10 @@
   private int numThreads = 0;
   /** If the connection is closed. */
   private boolean closed = false;
-
+  /** Last timestamp the connection was active. */
+  private long lastActiveTs = 0;
+  /** The connection's active status would expire after this window. */
+  private long activeWindow = TimeUnit.SECONDS.toMillis(30);
```

Comment:
`activeWindow` should be final static constant here?

Reply:
I agree. and called ACTIVE_WINDOW_TIME